### PR TITLE
Fixed Issue #306

### PR DESCRIPTION
**Pull request by anniesaxena**
## Related Issue

On clicking the Donate Us button on the homepage, it doesn't lead anywhere, the homepage window opens again.

Closes: #306 

### Describe the changes you've made

I added next link to donate-us and it now redirects to a /donate page where its shows that this feature will be available soon.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Tested on localhost

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly whereever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots

screencapture-localhost-3000-DonateUs-2021-04-26-22_38_14

### CODE REVIEW

**Comment by anniesaxena** [comment]
@aryasoni98 can you please review this

**Review by msk4862 — COMMENTED** [review]
@anniesaxena make sure all checks passes.

File: src/components/Home/HelpUs.jsx [hunk 1]
```diff
@@ -18,12 +14,7 @@ export default function HelpUs() {
           Let&apos;s help the society with free Education. We are on a mission
           of teaching 1 Lakh students for free.
         </p>
-        <button
-          className={styles.donateButton}
-          onClick={donateClick}
-          type="submit">
-          Donate Us
-        </button>
+        <Link href="/DonateUs"><a className={styles.donateButton}>Donate Us</a></Link>
```
Comment (aryasoni98) [inline src/components/Home/HelpUs.jsx:17]: Error: 17:32 error The href attribute is required for an anchor to be keyboard accessible. Provide a valid, navigable address as the href value. If you cannot provide an href, but still need the element to resemble a link, use a button and change it with appropriate styles.

**Review by aryasoni98 — COMMENTED** [review]
`/home/runner/work/Frontend/Frontend/src/components/Home/HelpUs.jsx`
10
Error:   17:32  error  The href attribute is required for an anchor to be keyboard accessible. Provide a valid, navigable address as the href value. If you cannot provide an href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md  jsx-a11y/anchor-is-valid

@anniesaxena

**Comment by anniesaxena** [comment]
@aryasoni98 Ii have made the requested changes.

**Review by msk4862 — APPROVED** [review]
Looks good to me :+1:

**Review by Anurag5086 — APPROVED** [review]
LGTM

**Review by aryasoni98 — APPROVED** [review]

**Pull request merged**